### ISSUE TITLE
Formatting

### ISSUE THREAD

**Issue by dolbi13** [maxnet/berryboot#608]
hello team.
i'm facing an issue with berry boot when i try to install it on a toshiba 1T HDD after the zeroing thing when he begins the partionning of the disk..after abt 2mn it says error partitionning any fix for this problem i can't figure it out :/

thanks for helping me out.

**Comment by actraiser30** [comment]
hi, make sure your hard disk does not have bad sectors you can check it with hard disk sentinel
also try the other format options eg: ext4.ext4 trim etc

**Comment by dolbi13** [comment]
yes i tried it but it doesn't work if i try to install it on my sdcard/usb drive it works fine but with the hdd drive no :/ is it because of it's size (1TB) ?

**Comment by actraiser30** [comment]
maybe the power supply is not enough for the hard drive connect it to a separate one
